include/internal/params.h

```diff
@@ -36,3 +40,17 @@ int ossl_param_get1_octet_string(const OSSL_PARAM *params, const char *name,
 int ossl_param_get1_concat_octet_string(const OSSL_PARAM *params, const char *name,
                                         unsigned char **out, size_t *out_len,
                                         size_t maxsize);
+
+/*
+ * Count the number of elements in a parameter list.
```

Comment:
single line comments please..

Reply:
Multi-line comments is the standard in this file and I've been required to change single line comments in a file that favoured multi-line comments before.  I don't mind changing this so long as nobody asks for it to be changed back…

Reply:
You re not following the coding standard